src/class-wpml-beaver-builder-register-strings.php

```diff
@@ -72,6 +76,11 @@ private function sort_modules_by_parent_and_child( array $all_modules, $parent_h
 			}
 		}
 
+		// Case when we have no hierarchical structure at the root (saved module)
+//		if ( null === $parent_hash && count( $all_modules ) !== count( $sorted_modules ) ) {
+//			return $all_modules;
+//		}
+
```

Comment:
@strategio is it really necessary to keep these comments here?

Reply:
Ooops, no it's a leftover, I will remove it, thanks!